server/src/main/java/org/apache/iotdb/db/metadata/MLogWriter.java

```diff
@@ -38,6 +41,8 @@
   private static final String STRING_TYPE = "%s,%s,%s"; 
   private File logFile;
   private BufferedWriter writer;
```

Comment:
You can use fileChannel to totally replace it.

Reply:
Fixed